Modules/Messages/src/Messages.Infrastructure/Persistence/IServiceCollectionExtensions.cs

```diff
@@ -22,6 +25,11 @@ public static void AddPersistence(this IServiceCollection services, PersistenceO
             blobOptions.RootFolder = options.BlobStorageOptions.Container);
         services.AddBlobStorage(options.BlobStorageOptions);
     }
+
+    public static void AddRepositories(this IServiceCollection services)
+    {
+        services.AddTransient<IMessagesRepository, MessagesRepository>();
+    }
```

Comment:
I know you copied that from the Tokens module, but I guess we should remove this method and move the logic to the `AddPersistence` method. I don't know why I created it back then.